client/rpcserver/types.go

```diff
@@ -68,11 +71,18 @@ type helpForm struct {
 	IncludePasswords bool   `json:"includepasswords"`
 }
 
+// tradeForm is information necessary to trade.
```

Comment:
Almost too vague.  How about "tradeForm combines the application password and the user's trade details" or something like that.  The docs you get from your IDE might as well let you know the intent of the type.